cddl_gen/cddl_gen.py

```diff
@@ -201,15 +204,16 @@ def from_cddl(cddl_class, cddl_string, default_max_qty, *args, **kwargs):
         # Parse the definitions, replacing the each string with a
         # CodeGenerator instance.
         for my_type, cddl_string in type_strings.items():
-            parsed = cddl_class(*args, default_max_qty, my_types, **kwargs, base_name=my_type)
+            parsed = cddl_class(*args, default_max_qty, my_types, **kwargs,
+                                base_name=my_type, base_stem=my_type)
```

Comment:
Passing the same value to two arguments looks wrong, should we explain why it makes sense here?

Reply:
Would this be something to handle within `cddl_class`? Such as `base_name=my_type)` here, then checking something like:

```
if base_type is None and base_name is not None:
    base_type = base_name
```

Reply:
This also comes down to being an initial value, which is passed down to sub-types, where they diverge in value.